### PR TITLE
Move updated bars to their own subscription and fix smaller issues

### PR DESCRIPTION
**Pull request by noramehesz**

### CODE REVIEW

File: lib/resources/datav2/websocket.ts [hunk 1]
```diff
@@ -165,6 +166,7 @@ export abstract class AlpacaWebsocket
     this.conn.binaryType = "nodebuffer";
     this.conn.once("open", () => this.authenticate());
     this.conn.on("message", (data: any) => {
+      console.log(this.msgpack.decode(data));
```
Comment (Mark-Pintye) [inline lib/resources/datav2/websocket.ts:169]: is this for debug or intended?

**Review by Mark-Pintye — APPROVED** [review]
lgtm

File: lib/resources/datav2/entityv2.ts [hunk 2]
```diff
@@ -357,6 +358,7 @@ export interface CryptoBar {
 export interface RawCryptoBar {
   T: string;
   S: string;
+  x: string;
```
Comment (drew887) [inline lib/resources/datav2/entityv2.ts:361]: Is this change in here to also [close out this PR](https://github.com/alpacahq/alpaca-trade-api-js/pull/190/files#r805775076)?

**Review by drew887 — APPROVED** [review]
lgtm but I do have the one question

File: lib/resources/datav2/entityv2.ts (hunk 2 above)
Comment (noramehesz) [reply lib/resources/datav2/entityv2.ts:361]: Yes, I will close that after this one will be merged and published.

**Pull request merged**